nixos/modules/services/networking/hylafax/options.nix

```diff
@@ -37,15 +35,15 @@ let
   modemConfigOptions = { name, config, ... }: {
     options = {
       name = mkOption {
-        type = str1;
+        type = nonEmptyStr;
```

Comment:
This seems like a slight semantic change, because `nonEmptyString` also matches strings only containing whitespace, but it's probably fine.